gsc/_front.scm

```diff
@@ -294,9 +292,10 @@
                          (compiler-user-warning
                           (source-locat (node-source ref))
                           (string-append
-                           "missing definition for `"
+                           "\""
```

Comment:
This could be written better to avoid writing a name that does not match the Scheme string syntax. This is also done in other places in the gsc source code, so this can wait until all places are changed together.

For example:

`(object->string (symbol->string '|abc\"def|))` => `"\"abc\\\"def\""`